packages/oui-field/src/field.controller.js

```diff
@@ -22,7 +22,8 @@ const VALIDATION_PARAMETERS = {
     min: ["min", "ng-min"],
     max: ["max", "ng-max"],
     minlength: ["minlength", "ng-minlength"],
-    maxlength: ["maxlength", "ng-maxlength"]
+    maxlength: ["maxlength", "ng-maxlength"],
+    custumValidation: ["pattern", "ng-pattern"]
```

Comment:
*custum* -> *custom*
Or could it be simply named "pattern"?

Reply:
+1 for pattern